src/core/utils/transactions.ts

```diff
@@ -642,7 +654,8 @@ export const getAdditionalDetails = (transaction: RainbowTransaction) => {
     !exchangeRate &&
     !collection &&
     !standard &&
-    !approval
+    !approval &&
+    contract?.name !== 'Rainbow'
```

Comment:
there's no other contract called Rainbow that could break this?

Reply:
@christianbaroni Do you know if that's the case ?